diagnostic.md

```diff
@@ -98,11 +126,24 @@ JSON.
 ```
 
 ```sh
-# your answer here
+- curl --include --request "PATCH" "http://localhost:3000/donuts/42" \
+-   --header "Content-Type: application/json" \
+-   --data $'{
+-     "donut": {
+-       "name": "Krüller",
+-     }
+-   }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function (form) {
+  let data = getFormFields(form);
+  return $.ajax({
+    url:  "http://localhost:3000" + "/donuts" + data.donut.id,
+    method: 'PATCH',
+    data: data,
+  });
+};
```

Comment:
If you used the form, you wouldn't need the `data.donut.id` You would only need `data.id` because it grabs the object from the form and then the ID value put in.